node/network/availability-distribution/src/requester/fetch_task.rs

```diff
@@ -0,0 +1,397 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
I'm generally very anti this module structure (`requester.rs`, `requester/x.rs`). Probably the biggest mistake in Rust 2018.

Reply:
Not sure I understand what you mean. What are your concerns exactly?

Reply:
I much prefer `requester/mod.rs`, `requester/x.rs` as all 'requester' logic appears close together in the file explorer.